stacks/patterns/index.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Title: Stacks
+ * Slug: stacks/index
+ * Categories: featured
+ */
+?>
+<!-- wp:group {"style":{"dimensions":{"minHeight":"100vh"},"position":{"type":"sticky","top":"0px"}},"backgroundColor":"white","layout":{"type":"flex","orientation":"vertical","verticalAlignment":"center","justifyContent":"center"}} -->
+<div class="wp-block-group has-white-background-color has-background" style="min-height:100vh"><!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group">
+
+<!-- wp:heading {"level":1,"fontSize":"x-large"} -->
+<h1 class="wp-block-heading has-x-large-font-size">Create Stacks.</h1>
+<!-- /wp:heading --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"style":{"dimensions":{"minHeight":"100vh"},"position":{"type":"sticky","top":"0px"},"color":{"background":"#f6f6f6"}},"layout":{"type":"flex","orientation":"vertical","verticalAlignment":"center","justifyContent":"center"}} -->
```

Comment:
Since the background colors of the slides are not part of the theme's palette, should we use something other than gray? this is a naive question from a non designer perspective, I'd just think it would look cooler. In the same way you added the image to the last slide, that made the whole theme much more solid, in my opinion. Would love other designer's eyes on this for feedback.